app/assets/stylesheets/sequence_builder.css

```diff
@@ -209,8 +202,40 @@ span.prefix{
   font-weight: bold;
 }
 
-/* SITE-WIDE ISH */
+/* SITE-WIDE BACKGROUND */
 
 body.ng-scope {
```

Comment:
This is probably not what you think it is. What were you trying to style with this?